runner.yml

```diff
@@ -324,7 +331,7 @@ Resources:
                   executor = "docker+machine"
                   [runners.docker]
                     tls_verify = false
-                    image = "ubuntu:latest"
+                    image = "alpine"
```

Comment:
Probably it'd be better to add another parameter with the default image name, to make it configurable on a per-stack basis.

Reply:
Added a string field with a default value for this one.